lib/beaker-hostgenerator/cli.rb

```diff
@@ -121,7 +121,7 @@ def initialize(argv = ARGV.dup)
         raise BeakerHostGenerator::Exceptions::SafeEarlyExit
       else
         # Tokenizing the config definition for great justice
-        @tokens = argv[0].split('-')
+        @tokens = __tokenize_input(argv[0])
```

Comment:
I don't know if the convention is as important as understandability for rubystas, so +1 on whatever naming scheme makes most sense to folks with more experience in ruby.